101-vm-simple-linux/README.md

```diff
@@ -1,4 +1,4 @@
-# Very simple deployment of a Linux VM
+# Linux Ubuntu 18.04-LTS
```

Comment:
This template will deploy whatever version the user selects, so this needs to be reverted or a more generic option chosen

Reply:
The original description says that will deploy a Ubuntu Linux VM with the default options being Standard_B2s size VM and the 18.04-LTS Version.

Reply:
# Very simple deployment of a Linux Ubuntu VM, perhaps?